### ISSUE TITLE
Append Cursor to bottom before writing

### ISSUE THREAD

**Issue by benhoff** [benhoff/CHATIMUSMAXIMUS#13]
Right now if I click in the text edit area, I'm pretty sure it moves the cursor, which messes up the append to bottom logic.

**Comment by benhoff** [comment]
Should be able to check if the cursor is at the end and then append to the bottom if not. Will need some in life testing.

**Comment by benhoff** [comment]
This will be fixed once 9180e690cb953426f46593d9f7f2729db61e86b5 is merged into master

**Comment by benhoff** [comment]
merged into master
